idc_index/index.py

```diff
@@ -177,6 +186,45 @@ def get_idc_version():
         idc_version = Version(idc_index_data.__version__).major
         return f"v{idc_version}"
 
+    def list_indices(self):
+        """
+        Lists all available indices including their installation status.
+
+        Returns:
+            indices_overview (pd.DataFrame): DataFrame containing information per index.
+        """
+
+        return pd.DataFrame.from_dict(self.indices_overview, orient="index")
+
+    def fetch_index(self, index) -> None:
```

Comment:
In this function I would not just fetch the file, but also load it into the class variable named as the index name - consistent with the main index.

Reply:
Done.